src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -2233,7 +2233,7 @@ internal unsafe void AnnounceColumnHeader(Point point)
                 lvhi.pt.X += si.nPos;
             }
 
-            if (User32.SendMessageW(hwnd, (User32.WM)HDM.HITTEST, IntPtr.Zero, ref lvhi) != (IntPtr)(-1) && lvhi.iItem > -1)
+            if (User32.SendMessageW(hwnd, (User32.WM)HDM.HITTEST, IntPtr.Zero, ref lvhi) != (IntPtr)(-1) && lvhi.iItem > -1 && IsAccessibilityObjectCreated)
```

Comment:
```suggestion
            if (IsAccessibilityObjectCreated && User32.SendMessageW(hwnd, (User32.WM)HDM.HITTEST, IntPtr.Zero, ref lvhi) != (IntPtr)(-1) && lvhi.iItem > -1)
```

There is no reason to send message if AO isn't created.